x-pack/plugins/security/kibana.json

```diff
@@ -3,7 +3,7 @@
   "version": "8.0.0",
   "kibanaVersion": "kibana",
   "configPath": ["xpack", "security"],
-  "requiredPlugins": ["features", "licensing"],
+  "requiredPlugins": ["features", "licensing", "status_page"],
```

Comment:
This was intentional, to ensure that `status_page` registered its anonymous route before `security` started.
However, upon re-reviewing the plugin lifecycle, I misunderstood one thing when we had created `status_page` -- for some reason I was under the impression that injected metadata could only be read in the `start` method, but it can actually be read in `setup`.

TL;DR no real need for this dependency, I'll make the change and remove it, thank you!